src/main/java/org/openhab/tools/analysis/checkstyle/ExportInternalPackageCheck.java

```diff
@@ -24,29 +24,29 @@
  */
 public class ExportInternalPackageCheck extends AbstractStaticCheck {
 
-    public final static String MANIFEST_EXTENSTION = "MF";
+    public final static String MANIFEST_EXTENSION = "MF";
     public final static String MESSAGE_INTERNAL_PACKAGE_EXPORTED = "Remove internal package export";
     public final static String MESSAGE_FILE_EMPTY = "File is empty !";
 
     public ExportInternalPackageCheck() {
-        setFileExtensions(MANIFEST_EXTENSTION);
+        setFileExtensions(MANIFEST_EXTENSION);
     }
 
     @Override
     protected void processFiltered(File file, List<String> lines) throws CheckstyleException {
         if (isEmpty(file)) {
-            log(0, MESSAGE_FILE_EMPTY, new Integer(0));
+            log(0, MESSAGE_FILE_EMPTY, 0);
```

Comment:
This is one of the first checks that we wrote and we wrote it against the Google checkstyle API (which is slightly different and I think it accepted only objects).